terraform/gitops/generate-files/templates/mojaloop/istio-config.yaml.tpl

```diff
@@ -435,4 +435,24 @@ spec:
         - destination:
             host: ${finance_portal_release_name}-reporting-hub-bop-shell
             port:
-              number: 80
\ No newline at end of file
+              number: 80
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: finance-portal-auth
+  namespace: ${istio_external_gateway_namespace}
+spec:
+  selector:
+    matchLabels:
+      app: ${istio_external_gateway_name}
+  action: CUSTOM
+  provider:
+    name: ${oathkeeper_auth_provider_name}
```

Comment:
You may need to add "oathkeeper_auth_provider_name" to mojaloop.tf file.

Reply:
oathkeeper_auth_provider_name  is already there.
@kalinkrustev, you want to move this to PR 75?

Reply:
Ok